content/messaging/customer-chat-api/v3.3/index.mdx

```diff
@@ -939,6 +941,196 @@ curl -X POST \
 
 </Section>
 
+## Config
+
+<Section>
+
+<Text>
+
+### Dynamic Config
+
+Returns the dynamic config of a given license.
+
+#### Specifics
+
+|                        |
+| ---------------------- | ----------------------------------------------------------------- |
+| **Method URL**         | `https://api.livechatinc.com/v3.3/customer/action/dynamic_config` |
+| **RTM API equivalent** | -                                                                 |
+| **Webhook**            | -                                                                 |
+
+#### Request
+
+| Parameters              | Required | Data type | Notes                                                       |
+| ----------------------- | -------- | --------- | ----------------------------------------------------------- |
+| `group_id`              | Yes      | `number`  | Id of the group that you want to get a dynamic config for.  |
+| `url`                   | No       | `string`  | The url that you want to get a dynamic config for.          |
+| `channel_type`          | No       | `string`  | The channel type that you want to get a dynamic config for. |
+| `test`                  | No       | `bool`    | Treats a dynamic config request as a test.                  |
+
+</Text>
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X GET \
+  'https://api.livechatinc.com/v3.3/customer/action/dynamic_config?license_id=<license_id>&group_id=0'
+```
+
+</CodeSample>
+
+<CodeResponse>
+
+```json
+{
+  "group_id": 0,
+  "client_limit_exceeded": false,
+  "domain_whitelisted": true,
+  "config_version": "194.159.276.6.1",
```

Comment:
have a random string as a response example, so that nobody has any wrong ideas about this value